.github/styles/templates/integration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Integrate <third-party tool name> with Timescale Cloud 
+excerpt: SEO friendly explanation of why the user will do it
+keywords: [<third-party tool name>]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+
+# Integrate <third-party tool name> with $CLOUD
+
+// Explain what the third-party tool is and what it does in their own words, link to product docs. 
+
+// Provide context for the integration steps, for example, if an additional connector is used.
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+- Install <third-party tool name> // Mention both server and cloud versions, if present. Link to installation pages.
+
+## Connect your $SERVICE_LONG
+
+Take the following steps to connect:
+
+<Procedure>
+
+1. **Log in to <third-party tool name>**
+
+    // Sub-steps, code, screenshots if necessary.
+
+1. **Configure the connection**
+
+   // Sub-steps, code, screenshots if necessary. Link to [Find your connection details][connection-info].
+
+   ...
+
+1. **Test the connection**
+
+   // Sub-steps, code, screenshots if necessary.
+
+</Procedure>
+
+## Test the integration 
```

Comment:
Not sure test is the right word, as the previous section already has a test step. 

<Do something> with <third party tool> and $CLOUD_LONG maybe. 

Say only if there is an easy good example for this.